packages/design-system/src/styles/components/_Button.scss

```diff
@@ -182,21 +182,37 @@
 %transparent-inverse-button {
   border-color: transparent;
   background-color: transparent;
+
+  &:disabled,
+  &.ds-c-button--disabled {
+    border-color: transparent;
+    background-color: transparent;
+  }
+  &:hover,
+  &.ds-c-button--hover {
+    color: rgba($color-base-inverse, 0.8);
+  }
+
   // Default inverse focus styles are overridden above
   @if $ds-include-focus-styles {
     &:focus,
     &.ds-c-button--focus {
       border-color: transparent;
+      background-color: $focus-color;
```

Comment:
Same here, isnt this updating the styles for the beta focus styles? Why is this being changed

Reply:
oh ok I think I understand, is this addition necessary because we updated the ordering for active to come after focus for the transparent inverse button?

Reply:
I was switching between the focus styles on and off in the PR to make sure i did not cause any regressions and noticed an issue with the transparent inverse button so decided to put it in this PR. 

**I can back those out of this PR and put them into a different one if you like** 

Screenshot is what the issue was 
![Screen Shot 2021-02-24 at 1 34 20 PM](https://user-images.githubusercontent.com/1449852/109048372-01fd7000-76a5-11eb-9ef3-19a9d8e2fa06.png)

Reply:
no this is a great catch thank you